src/reset-password/tests/ResetPasswordPage.test.jsx

```diff
@@ -157,6 +157,55 @@ describe('ResetPasswordPage', () => {
 
   // ******** miscellaneous tests ********
 
+  it('check password value on blur and enable show password field', () => {
+    const resetPasswordPage = mount(reduxWrapper(<IntlResetPasswordPage {...props} />));
+    resetPasswordPage.find('input#newPassword').simulate('change', { target: { value: 'aziz156@', name: 'newPassword' } });
+    resetPasswordPage.find('input#newPassword').simulate('blur', { target: { value: 'aziz156@', name: 'newPassword' } });
+    expect(resetPasswordPage.find('input#newPassword').props().value).toEqual('aziz156@');
+  });
```

Comment:
Could you elaborate this test? What we are doing in this test?

Reply:
addressed